### ISSUE TITLE
ValueError initializing pages

### ISSUE THREAD

**Issue by dankeemahill** [jsvine/pdfplumber#41]
I get a `ValueError: Cannot convert <PDFObjRef:4> to Decimal.` when accessing the pages of [this pdf](https://github.com/openelections/openelections-sources-tx/raw/master/2016/Hays%20TX%2011-8-2016%2Bhays%2Bcounty%2Btotal%2Bcanvass.pdf) with pdfplumber==0.5.5.

I notice the `pdfplumber.open` call seems to run much quicker on this file compared to other files that don't raise this error.

```python
pdf = pdfplumber.open('Hays TX 11-8-2016+hays+county+total+canvass.pdf')
pdf.pages
```
⬇️ 
```
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-9-bc19284d68b8> in <module>()
      1 pdf = pdfplumber.open('Hays TX 11-8-2016+hays+county+total+canvass.pdf')
----> 2 pdf.pages

/anaconda3/lib/python3.6/site-packages/pdfplumber/pdf.py in pages(self)
     54             page_number = i+1
     55             if pp != None and page_number not in pp: continue
---> 56             p = Page(self, page, page_number=page_number, initial_doctop=doctop)
     57             self._pages.append(p)
     58             doctop += p.height

/anaconda3/lib/python3.6/site-packages/pdfplumber/page.py in __init__(self, pdf, page_obj, page_number, initial_doctop)
     21 
     22         cropbox = page_obj.attrs.get("CropBox", page_obj.attrs.get("MediaBox"))
---> 23         self.cropbox = self.decimalize(cropbox)
     24 
     25         if self.rotation in [ 90, 270 ]:

/anaconda3/lib/python3.6/site-packages/pdfplumber/page.py in decimalize(self, x)
     39 
     40     def decimalize(self, x):
---> 41         return utils.decimalize(x, self.pdf.precision)
     42 
     43     @property

/anaconda3/lib/python3.6/site-packages/pdfplumber/utils.py in decimalize(v, q)
     87             return Decimal(repr(v))
     88     else:
---> 89         raise ValueError("Cannot convert {0} to Decimal.".format(v))
     90 
     91 def is_dataframe(collection):

ValueError: Cannot convert <PDFObjRef:4> to Decimal.
```

**Comment by jsvine** [comment]
Big thanks for flagging, Dan! Just pushed a fix, which should now be available in `v0.5.6`. The PDF you've referenced now seems to load fine for me. Let me know if you find otherwise.

**Comment by dankeemahill** [comment]
I also was able to init pages on the referenced PDF with `v0.5.6`. Thanks! ✅
